tchannel/testing/vcr/config.py

```diff
@@ -141,8 +146,43 @@ def test_bar():
         If True, when ``use_cassette`` is used as a decorator, the cassette
         object will be injected into the function call as the first argument.
         Defaults to False.
+    :param matchers:
+        Used to configure the request attributes which VCR matches on. This is
+        a list of request attributes or a function that accepts a list of
+        request attributes and returns a new list of attributes. This function
+        will be called with the default list of attributes used by VCR and the
+        result will be the list of matchers used. A recorded response will be
+        replayed if all specified attributes of the corresponding request
+        match the request that is being made. Valid attributes are:
+        ``serviceName``, ``hostPort``, ``endpoint``, ``headers``, ``body``,
+        ``argScheme``, and ``transportHeaders``.
+
+        For example,
+
+        .. code-block:: python
+
+            @vcr.use_cassette('tests/data/foo.yaml', matchers=['body']):
+            def test_foo():
+                # ...
+
+            def no_headers(matchers):
```

Comment:
That wouldn't allow adding things that aren't already in the default list of matchers